GithubPractice/ViewController.swift

```diff
@@ -26,6 +26,7 @@ class ViewController: UIViewController, UITextFieldDelegate {
         jobPositionField.delegate = self
         
         saveButton.layer.cornerRadius = 8
+        clearButton.layer.cornerRadius = 8
```

Comment:
if all the highlight buttons have same corner radius in the app, better set it in the HighlightButton class

Reply:
okay